### ISSUE TITLE
Visualize how much training is left

### ISSUE THREAD

**Issue by olinesn** [murthylab/sleap#624]
It might be possible to quickly visualize at a glance roughly how much training is left by reporting the number of epochs since the Best Model, and/or as a fraction relative to Training Patience. The Best Model could also be marked on the plot with a red dot. This way, one could quickly get a feel for if it makes sense to stop training early or not.

Plotting the Validation Loss with black markers might also help with discriminability, I've illustrated this idea below.
![Training_delay](https://user-images.githubusercontent.com/69451171/147722606-6741a334-4a30-49a3-8f4d-f4dbae0893a1.png).

Thanks!

**Comment by Rodchi** [comment]
is there a way of getting this image on a folder after the training is over?

**Comment by talmo** [comment]
Hi @TheMalaland,

We don't save this image, but you can replot the loss curves using the values stored in `training_log.csv` in the model folder.

Cheers,

Talmo
